src/style.css

```diff
@@ -0,0 +1,112 @@
+@import url(./style_stars.css);
+
+* {
+  margin: 0;
+  text-decoration: none;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  background: #f3f3f3;
+
+  --card-color-text: #616070;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 198px;
+  height: 406px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-img {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 0;
+}
+
+.card__name-product {
+  width: 166px;
+  height: 36px;
+  padding: 40px 16px 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code-product {
+  width: 98px;
+  height: 14px;
+  padding: 4px 16px 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--card-color-text);
+}
+
+.card__product-review {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 16px 16px 0;
+}
+
+.card__number-of-review {
+  display: flex;
+  align-items: center;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--card-color-text);
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 24px 16px 0;
+}
+
+.card__price-text {
+  width: 31px;
+  height: 18px;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--card-color-text);
+}
+
+.card__price-price {
+  width: 50px;
+  height: 18px;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--card-color-text);
+}
+
+.card__buttom {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
```

Comment:
[It](https://github.com/mate-academy/layout_product-cards/pull/2203#discussion_r1027079044) means for all colors. Be consistent ;)